modules/openy_gc_storage/openy_gc_storage.install

```diff
@@ -311,6 +311,21 @@ function openy_gc_storage_update_8016() {
  * Update/create configs related to Duration reference field.
  */
 function openy_gc_storage_update_8017() {
+  // Ensure duplicate content doesn't exist before creating the field.
+  $database = \Drupal::database();
+  $tables = [
+    'taxonomy_term__field_gc_duration_max',
+    'taxonomy_term__field_gc_duration_min',
+    'taxonomy_term_revision__field_gc_duration_max',
+    'taxonomy_term_revision__field_gc_duration_min',
+  ];
+
+  foreach ($tables as $table) {
+    if ($database->schema()->tableExists($table)) {
+      $database->truncate($table);
```

Comment:
@podarok that makes sense. What if we just leave these here but commented out? That way in the rare case that the hook breaks we're leaving people the option to fix it, but otherwise we don't change anything.

Reply:
Best way is to have a drush command for these types of fixes and document them @froboy

Reply:
Can we instead of just commenting out, build hook_requirements() and check if this hook must be run? @froboy